cores/esp8266/WString.cpp

```diff
@@ -98,6 +98,59 @@ String::String(unsigned long value, unsigned char base) {
     *this = buf;
 }
 
+String::String(long long value) {
+    init();
+    char buf[2 + 8 * sizeof(long long)];
+    sprintf(buf, "%lld", value);
+    *this = buf;
+}
+
+String::String(unsigned long long value) {
+    init();
+    char buf[1 + 8 * sizeof(unsigned long long)];
+    sprintf(buf, "%llu", value);
+    *this = buf;
+}
+
+#if 1
```

Comment:
I'd still drop the `#if 1` bit and just remove the `charconv` portion to avoid `ifdef` hell.  The git commit will reference this chain, so if someone really cares they can go back and find what testing you did to end up here.

Maybe just a note that "using charconv is possible but cost 880b of flash and 256b of RAM so was not acceptable" or something in the comments if you really want to be explicit.

I can't see String(int64) constructor performance being a concern to people.  If they cared about performance they wouldn't be using String in the first place...

Reply:
Of course that was intended :]